modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1902,6 +1902,63 @@ public void testAddOrUpdateObjectEntryWithObjectDefinitionTree()
 			_objectEntryLocalService);
 	}
 
+	@Test
+	public void testAddSystemObjectEntryWithObjectValidationRule()
+		throws Exception {
+
+		// Old value
```

Comment:
// can only update the user if the email address is equal to a specific value
:thinking:

Reply:
Using the OldValue() function in the ValidationRule means: I can only update an object entry if the field X equals the value Y.

Reply:
Hi, I just made it a little shorter because it was too long for SF